x-pack/test/plugin_api_integration/test_suites/licensed_feature_usage/feature_usage.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+
+  const notifyUsage = async (featureName: string, usedAt: number) => {
+    await supertest.get(`/api/feature_usage_test/hit?featureName=${featureName}&usedAt=${usedAt}`);
+  };
+
+  const clearUsages = async () => {
+    await supertest.get(`/api/feature_usage_test/clear`);
+  };
+
+  const toISO = (time: number) => new Date(time).toISOString();
+
+  describe('/api/licensing/feature_usage', () => {
+    beforeEach(async () => {
+      await clearUsages();
+    });
+
+    it('returns a map of last feature usages', async () => {
+      const timeA = Date.now();
+      await notifyUsage('test_feature_a', timeA);
```

Comment:
Do we need `await` on the function call?

Reply:
I think we do, else we wont necessarily have the call finished before querying `/api/licensing/feature_usage` down in the test?

Reply:
but it seems to be a sync function. isn't it? https://github.com/elastic/kibana/pull/63549/files#diff-6e29a9240780490ad9d109842008ec2eR25

Reply:
my bad, I missed it's a remote call